### ISSUE TITLE
All the icons in the taskbar are not fitting

### ISSUE THREAD

**Issue by Nukethefox** [ChrisAnd1998/TaskbarXI#158]
When having more than 21 icons in my taskbar, aligned to the center, in a 1920x1080 screen (with "scale" set to 125%), with both the tray and the traywnd hidden, the icons in the left are not fitting. You can see the screenshot, with the windows button not showing. 

![Captura de pantalla 2023-02-18 120541](https://user-images.githubusercontent.com/125735099/219856948-7cd9abba-82d7-46a6-89ab-ba7ebc78126b.png)

When I align it to the left tho, all of them fit perfectly. I had to delete some of the icons from the taskbar, so I can have it in the center without some of them hiding.

Also, more people mentioned it, but having "auto-hide taskbar" enabled, when I put the mouse in the bottom of the screen, the taskbar won't show. Hope it gets fixed soon.

I'm in Windows 11 of course, 22H2, 22621.1265. This is the configuration I'm using in TaskbarXI:

![Captura de pantalla 2023-02-18 121020](https://user-images.githubusercontent.com/125735099/219857150-dd73805e-9083-4037-9f18-5ec76a1261ad.png)

**Comment by DamienAmbrose** [comment]
Yep, it's still there. For some reason, even if the tray window's hidden, TaskbarXI still treats it like it's still there and adds some phantom padding to the icons themselves. 
![image](https://github.com/ChrisAnd1998/TaskbarXI/assets/89913900/ce939f48-68ba-41ff-9d3b-16d5c2d44952)
![image](https://github.com/ChrisAnd1998/TaskbarXI/assets/89913900/d3383d94-d2a1-4486-81be-076c81b6f9a6)

If this is an issue with Windows itself, I don't think there's much Chris can do other than shifting the whole taskbar to the left by resizing it (which would still be better than cropping the icons out, mind you).
Or maybe just don't open 21 apps at once. Virtual desktops exist. You're welcome.
